.devfile

```diff
@@ -0,0 +1,24 @@
+specVersion: 0.0.1
+name: che-in-che
+projects:
+  - name: che
+    source:
+      type: git
+      location: 'https://github.com/eclipse/che.git'
+tools:
+  - name: theia-editor
+    type: cheEditor
+    id: org.eclipse.che.editor.theia:1.0.0
+  - name: exec-plugin
+    type: chePlugin
+    id: che-machine-exec-plugin:0.0.1
+  - name: che-dev
+    type: kubernetes
+    local: .che-dev.yaml
+commands:
+  - name: build
+    actions:
+      - type: exec
+        tool: che-dev
+        command: cd /projects/che && mvn -Pnative clean install
```

Comment:
I'm OK with merging this PR as an initial state. Just FYI command is failed with the following error
![screenshot_20190204_171022](https://user-images.githubusercontent.com/5887312/52216849-e0ec3b80-289f-11e9-9bb9-d5bd0fcd5c76.png)

Reply:
I'll take a look.